src/Raven.Studio/typescript/models/database/tasks/periodicBackup/s3Settings.ts

```diff
@@ -68,18 +82,44 @@ class s3Settings extends amazonSettings {
                 }
             ]
         });
+        
+        this.customServerUrl.extend({
+            required: {
+                onlyIf: () => this.useCustomS3Host()
+            },            
+            validation: [
+                {
+                    validator: (customServerUrl: string) => !this.useCustomS3Host() ||
+                                                            (!this.hasConfigurationScript() && customServerUrl && genUtils.urlRegex.test(customServerUrl)),                    
+                    message: "Expected format: 'http(s)://<yourHostAddress>'"
```

Comment:
I think we have different message when url is invalid.

Reply:
ok - will change to be the same